src/ui/preferences.ui

```diff
@@ -298,12 +298,33 @@ the application.</string>
             </item>
            </layout>
           </item>
+          <item>
+           <layout class="QHBoxLayout" name="horizontalLayout_19">
+            <item>
+             <widget class="QLabel" name="label_30">
+              <property name="text">
+               <string>Interface Font Size</string>
+              </property>
+             </widget>
+            </item>
+            <item>
+             <widget class="QSpinBox" name="interfaceFontSize">
+              <property name="minimum">
+               <number>8</number>
+              </property>
+              <property name="maximum">
+               <number>32</number>
+              </property>
```

Comment:
The range could be bigger? The MS Word's default font list is `[5,72]`.
<img width="30" src="https://github.com/user-attachments/assets/de94512f-c48d-47a5-924e-6803c2b4537a" />